sql/schema.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE author (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    first_name VARCHAR,
+    last_name VARCHAR
+);
+
+
+CREATE TABLE source (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR,
+);
+
+CREATE TABLE book(
+  id SERIAL PRIMARY KEY,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  publisher VARCHAR(30) NOT NULL,
+  cover_state VARCHAR(10) NOT NULL,
+  label_id INT,
+  author_id INT,
```

Comment:
- [ ] As the project requirements documentation suggests, you are supposed to create all the tables and relationships as represented in the given [UML diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png)
After taking a look at your schema.sql, I found that you have not created or maybe you have not listed the `musicAlbum` table and the `genre` table while they are essential. I would suggest that you create them and/or add them in the `schema.sql` file and their associations if there are any

Reply:
Done!